ROADMAP.md

```diff
@@ -47,18 +43,31 @@ more IPAM strategies besides subnet per Node, like IP pool per Node or
 per Namespace.
 
 * **Egress policy**
-Egress policy is to control the egress Nodes and SNAT IPs of traffic from Pods
-to external network. For example, a user can request a dedicated SNAT IP per
-Namespace, or for a particular set of Pods or Services. This feature is very
-useful for services in the Node or external network to identify the source of
-Pod traffic based on SNAT IP and enforce specific policies on the traffic. Also
-check the [egress policy proposal](https://github.com/vmware-tanzu/antrea/issues/667)
-to learn more.
+Antrea released alpha support for [Egress](docs/feature-gates.md#egress) in
+version 1.0.0. User can specify a specific SNAT IP for a selected set of Pods
+with an Egress CRD, and then the egress traffic from the Pods to external
+network will be SNAT'd using the SNAT IP. This feature is very useful for
+services in the Node or external network to identify the source of Pod traffic
+based on SNAT IP and enforce specific policies on the traffic. However, the
+Egress feature still has several major limitations to be addressed. For example,
```

Comment:
s/to be addressed/which need to be addressed

Reply:
Done